test/unit/assorted/auth.spec.test.ts

```diff
@@ -1,13 +1,19 @@
 import { loadSpecTests } from '../../spec';
 import { executeUriValidationTest } from '../../tools/uri_spec_runner';
 
+const SKIP = ['should throw an exception if username and no password (MONGODB-AWS)'];
```

Comment:
This should have been implemented as part of the closed NODE-2288 ticket but wasn't

Reply:
https://jira.mongodb.org/browse/NODE-3986